liblangutil/Scanner.cpp

```diff
@@ -683,6 +684,14 @@ void Scanner::scanToken()
 					else
 						token = setError(ScannerError::IllegalToken);
 				}
+				else if (token == Token::Unicode)
+				{
+					// Special quoted hex string must follow
```

Comment:
Don't have the full context, but shouldn't we also reset `m` and `n` here?

Reply:
Fixed.